### DISCUSSION TITLE
Concat in form

### DISCUSSION THREAD

**Discussion by Dev-Lopez02** [serenity-is/Serenity#6521] [Serenity Ideas]
has anyone done a concat with substring in a form?

I have looked for thousands of solutions and I still can't find anything, I want to do something like this, I have two forms, in the first one I record a code and a description, when I open the other form, I look for the first code and when I enter the second code, I would like to automatically type the code that I select in the combo box.

-1
>1(the code of the first form)200

**Comment by LSNicholls** [comment]
Can you show a screenshot of what you are looking for?  I may actually have done this but I may be misunderstanding what you want.
